plugins/source/slack/client/multiplexers.go

```diff
@@ -0,0 +1,12 @@
+package client
+
+import "github.com/cloudquery/plugin-sdk/schema"
+
+func TeamMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
+	client := meta.(*Client)
+	clients := make([]schema.ClientMeta, 0, len(client.Teams))
+	for i, team := range client.Teams {
+		clients[i] = client.withTeamID(team.ID)
```

Comment:
prealloc above makes the len = 0, so this will panic: https://go.dev/play/p/O5TBscjbVZU

Reply:
```suggestion
		clients = append(clients, client.withTeamID(team.ID))
```

Reply:
Updated 👍  I'll still do some more testing before merging anyway